src/etc/inc/ipsec.inc

```diff
@@ -1466,7 +1466,15 @@ function ipsec_setup_pools() {
 		return;
 	}
 	$scconf['pools']['mobile-pool'] = array();
+	$pool_addrs = array();
 	$pool_common =& $scconf['pools']['mobile-pool'];
+	if (!empty($a_client['pool_address'])) {
+		$pool_addrs[] = "{$a_client['pool_address']}/{$a_client['pool_netbits']}";
+	}
+	if (!empty($a_client['pool_address_v6'])) {
+		$pool_addrs[] = "{$a_client['pool_address_v6']}/{$a_client['pool_netbits_v6']}";
+	}
+	$scconf['pools']['mobile-pool']['addrs'] = implode(',', $pool_addrs);
```

Comment:
This still needs a check to ensure it isn't added with an empty value. And the variable reference didn't get changed.

So it should be:

    if (!empty($pool_addrs)) {
    	$pool_common['addrs'] = implode(',', $pool_addrs);
    }

Reply:
oh, ok, fixed